.vib/redis/goss/goss.yaml

```diff
@@ -0,0 +1,64 @@
+{{- $auth := printf "REDISCLI_AUTH=%s" .Vars.auth.password }}
+{{- $master_endpoint := printf "-h redis-master -p %d" .Vars.master.service.ports.redis }}
+{{- $replicas_endpoint := printf "-h redis-replicas -p %d" .Vars.replica.service.ports.redis }}
+{{- $replicas := .Vars.replica.replicaCount }}
+command:
+  {{- $key := printf "key_%s" (randAlpha 5) }}
+  {{- $value := printf "value_%s" (randAlpha 5) }}
+  redis-set-key-value-pairs:
+    exec: export {{ $auth }} && redis-cli {{ $master_endpoint }} SET {{ $key }} {{ $value }} {{ range $e, $i := until $replicas }} && redis-cli -h redis-replicas-{{ $i }}.redis-headless -p {{ $.Vars.replica.containerPorts.redis }} GET {{ $key }} | grep -q {{ $value }}{{ end }}
+    exit-status: 0
+  redis-disabled-command:
+    exec: {{ $auth }} redis-cli {{ $master_endpoint }} {{ .Vars.master.disableCommands }}
+    exit-status: 0
+    stdout:
+      - ERR unknown command
```

Comment:
I would probably use a loop here, as `disableCommands` is originally an array.

Reply:
Good catch!